src/Compilers/CSharp/Portable/PublicAPI.Unshipped.txt

```diff
@@ -182,6 +183,7 @@ abstract Microsoft.CodeAnalysis.CSharp.Syntax.CommonForEachStatementSyntax.ForEa
 abstract Microsoft.CodeAnalysis.CSharp.Syntax.CommonForEachStatementSyntax.InKeyword.get -> Microsoft.CodeAnalysis.SyntaxToken
 abstract Microsoft.CodeAnalysis.CSharp.Syntax.CommonForEachStatementSyntax.OpenParenToken.get -> Microsoft.CodeAnalysis.SyntaxToken
 abstract Microsoft.CodeAnalysis.CSharp.Syntax.CommonForEachStatementSyntax.Statement.get -> Microsoft.CodeAnalysis.CSharp.Syntax.StatementSyntax
+override Microsoft.CodeAnalysis.CSharp.CSharpCompilation.GetRequiredLanguageVersion(Microsoft.CodeAnalysis.Diagnostic diagnostic) -> string
```

Comment:
Yes, it is odd. But I don't have much choice. The `LanguageVersion` enum is language-specific, but this API is defined on `Compilation` which is language-independent.